new-lamassu-admin/src/pages/Customers/components/PhotosCard.js

```diff
@@ -0,0 +1,167 @@
+import ButtonBase from '@material-ui/core/ButtonBase'
+import Paper from '@material-ui/core/Card'
+import { makeStyles } from '@material-ui/core/styles'
+import * as R from 'ramda'
+import React, { memo, useState } from 'react'
+import Carousel from 'react-material-ui-carousel'
+
+import { InformativeDialog } from 'src/components/InformativeDialog'
+import { Info2, Label1 } from 'src/components/typography'
+import { ReactComponent as CrossedCameraIcon } from 'src/styling/icons/ID/photo/crossed-camera.svg'
+import { URI } from 'src/utils/apollo'
+
+import CopyToClipboard from '../../Transactions/CopyToClipboard'
+
+import styles from './PhotosCard.styles'
+
+const useStyles = makeStyles(styles)
+
+const Label = ({ children }) => {
+  const classes = useStyles()
+  return <Label1 className={classes.label}>{children}</Label1>
+}
+
+const PhotosCard = memo(({ frontCameraData, txPhotosData }) => {
+  const classes = useStyles()
+
+  const [photosDialog, setPhotosDialog] = useState(false)
+
+  const mapKeys = pair => {
+    const key = R.head(pair)
+    const value = R.last(pair)
```

Comment:
Can't this just use a regular JS deconstruction? 
```suggestion
    const [key, value] = pair
```